projects/packages/my-jetpack/src/products/class-social.php

```diff
@@ -0,0 +1,131 @@
+<?php
+/**
+ * Search product
+ *
+ * @package my-jetpack
+ */
+
+namespace Automattic\Jetpack\My_Jetpack\Products;
+
+use Automattic\Jetpack\My_Jetpack\Hybrid_Product;
+use Automattic\Jetpack\My_Jetpack\Wpcom_Products;
+
+/**
+ * Class responsible for handling the Social product
+ */
+class Social extends Hybrid_Product {
+
+	/**
+	 * The product slug
+	 *
+	 * @var string
+	 */
+	public static $slug = 'social';
+
+	/**
+	 * The Jetpack module name
+	 *
+	 * @var string
+	 */
+	public static $module_name = 'social';
```

Comment:
Just a question, but is this supposed to be an actual Jetpack module (like Publicize)?